### PR TITLE
Hrosenbe/weat 14 prepare only with cassandra

### PR DESCRIPTION
**Pull request by hrosenbe**
This pull request replaces MongoDB with Cassandra for the event and image store.

### CODE REVIEW

File: auctionBidService/src/main/java/com/vmware/weathervane/auction/data/imageStore/NoBenchmarkInfoException.java~2e48a52... Removed the notion and code supporting database scales [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*
```
Comment (bhoflich) [inline auctionBidService/src/main/java/com/vmware/weathervane/auction/data/imageStore/NoBenchmarkInfoException.java~2e48a52... Removed the notion and code supporting database scales]: There are a number of 'alternate version' files in this directory. (~HEAD, ~2e48a52...)
Are they needed?

File: build.gradle [hunk 2]
```diff
@@ -52,6 +52,7 @@ configure(subprojects.findAll {it.name == 'auctionApp' || it.name == 'auctionBid
 		postgreSQLJdbcVersion = "9.4.1212.jre7"
 		dbcpVersion = "1.4"
 		springDataMongoVersion = "1.10.9.RELEASE"
```
Comment (bhoflich) [inline build.gradle]: Should the mongo references in this file be removed?

**Comment by hrosenbe** [comment]
I didn't bother with the License file as it needs to be updated and all references to other licenses removed.

File: auctionBidService/src/main/java/com/vmware/weathervane/auction/data/imageStore/NoBenchmarkInfoException.java~2e48a52... Removed the notion and code supporting database scales (hunk 1 above)
Comment (hrosenbe) [reply auctionBidService/src/main/java/com/vmware/weathervane/auction/data/imageStore/NoBenchmarkInfoException.java~2e48a52... Removed the notion and code supporting database scales]: I have never seem that before.  I have removed the files.

File: build.gradle (hunk 2 above)
Comment (hrosenbe) [reply build.gradle]: Yes.  I have updated this file.

File: weathervane.config.docker [hunk 3]
```diff
@@ -35,14 +35,14 @@
...
-  "numAppServers" : 3,
+  "numAppServers" : 1,
   "appServerHosts" : [ "AuctionApp1", "AuctionApp2", "AuctionApp3", ],
   "appServerCpus" : "1",
   "numMsgServers" : 1,
   "msgServerHosts" : [ "AuctionMsg1", ],
   "msgServerCpus" : "1",
   "numCoordinationServers" : 3,
-  "coordinationServerHosts" : [ "AuctionCs1", "AuctionCs3", "AuctionCs3", ],
+  "coordinationServerHosts" : [ "AuctionCs1", "AuctionCs2", "AuctionCs3", ],
```
[the remark this answers is not in the document]
Comment (hrosenbe) [reply weathervane.config.docker]: Good catch.  Fixed.

**Pull request merged**